release-notes/opensearch-dashboards.release-notes-1.3.8.md

```diff
@@ -0,0 +1,32 @@
+# Version 1.3.8 Release Notes
+
+
+### 🛡 Security
+
+- [CVE-2022-25901] Bump supertest ([#3326](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3326))
+- [CVE-2022-25860] Bumps simple-git from 3.15.1 to 3.16.0 ([#3345](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3345))
```

Comment:
Can we update all `Bumps` to `Bump`? It is better to modify a bit to make it consistent with other release note, ex https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-2.5.0.md

Reply:
Thanks for the comments! Address it as per suggestion.